test/features/fixture_data/help_commands.txt

```diff
@@ -2,12 +2,14 @@
 | Command     | Shortcut          | Description                                             |
 +-------------+-------------------+---------------------------------------------------------+
 | \G          | \G                | Display results vertically.                             |
+| \d          | \d [table]        | List tables or show create table.                       |
 | \dt         | \dt [table]       | List or describe tables.                                |
 | \e          | \e                | Edit command with editor. (uses $EDITOR)                |
 | \f          | \f [name]         | List or execute favorite queries.                       |
 | \fd         | \fd [name]        | Delete a favorite query.                                |
 | \fs         | \fs name query    | Save a favorite query.                                  |
 | \l          | \l                | List databases.                                         |
+| \once       | \o [-o] filename  | Output for the next SQL command only to FILENAME        |

```

Comment:
This command was included in a recent commit, but wasn't added to this file, I figured I might as well add it? (And by that, I really mean, I just copied and pasted this output from running `\?`, saw this change, and figured it might as well go in?)